app/models/message.rb

```diff
@@ -0,0 +1,16 @@
+class Message < ApplicationRecord
+  belongs_to :sender, :foreign_key => :sender_id, class_name: 'User'
+  belongs_to :recipient, :foreign_key => :recipient_id, class_name: 'User'
+  
+  has_many :contents, dependent: :destroy
+  
+  validates_uniqueness_of :sender_id, :scope => :recipient_id
+  
+  scope :between, -> (sender_id, recipient_id) do
+    where("(messages.sender_id = ? AND messages.recipient_id =?) OR (messages.sender_id = ? AND messages.recipient_id =?)", sender_id, recipient_id, recipient_id, sender_id)
```

Comment:
The variable names are confusing with how this method is currently written. Could you change this to use variable names like `user_one` and `user_two` instead of `sender_id` and `recipient_id` since you're checking the ids for being either the sender or the recipient?